pkg/virt-controller/watch/topology/tsc_test.go

```diff
@@ -56,22 +56,31 @@ var _ = Describe("TSC", func() {
 		),
 		Entry(
 			"on a scalable node where not all required frequencies are compatible",
-			[]int64{1, 2, 3, 200},
+			[]int64{1, 2, 3, 123130, 200000}, // 123130 is above but within 250 PPM
 			[]int64{2, 4},
-			int64(123),
+			int64(123123),
 			true,
-			[]int64{1, 2, 3, 123},
+			[]int64{1, 2, 3, 123123, 123130},
 			[]int64{4},
```

Comment:
Isn't `4` a valid frequency in this case?

Reply:
`4` would be valid, but it's in the list of frequencies that currently have a label on the node but no VMI matching it.
Since it's neither the frequency of the node nor the frequency of an existing VMI, the label for that frequency is expected to get removed.